netmiko/base_connection.py

```diff
@@ -1320,6 +1321,20 @@ def find_prompt(
                     self.write_channel(self.RETURN)
```

Comment:
i dont think this way of sending "\n" multiple times is correct. a single "\n" should be enough for us to find the prompt. we should use the find_method in vxr_ssh